Add docs for the image pull secrets CO option (#1581)
strimzi/strimzi-kafka-operator: documentation/book/ref-operators-cluster-operator-configuration.adoc

The valid values are `Always`, `IfNotPresent`, and `Never`.
If not specified, the {ProductPlatformName} defaults will be used.
Changing the policy will result in a rolling update of all your Kafka, Kafka Connect, and Kafka Mirror Maker clusters.

`STRIMZI_IMAGE_PULL_SECRETS`:: Optional.
A comma-separated list of `Secret` names.

> The secrets referenced here should contain the credentials to the container registries where the container images should be pulled from.

Reviewer: ```suggestion
The secrets referenced here contain the credentials to the container registries where the container images are pulled from.
```

> The secrets will be used in the `imagePullSecrets` field for all `Pods` created by the Cluster Operator.

Reviewer: ```suggestion
The secrets are used in the `imagePullSecrets` field for all `Pods` created by the Cluster Operator.
```

> Changing this list will result in a rolling update of all your Kafka, Kafka Connect, and Kafka Mirror Maker clusters.

Reviewer: ```suggestion
Changing this list results in a rolling update of all your Kafka, Kafka Connect, and Kafka Mirror Maker clusters.
```
